### PR TITLE
[españolQue] Fix code coverage

### PR DESCRIPTION
**Pull request by willjvsmith**
* Adds test for code coverage

PR 100 get in

### CODE REVIEW

File: src/test/scala/temple/utils/StringUtilsTest.scala [hunk 1]
```diff
@@ -59,4 +59,10 @@ class StringUtilsTest extends FlatSpec with Matchers {
   it should "correctly convert a capitalised phrase" in {
     snakeCase("CustomerID") shouldBe "customer_id"
   }
+
+  behavior of "españolQue"
+
+  it should "wrap a string as a Spanish question" in {
+    españolQue("Are we having paella tonight") shouldEqual ("¿Are we having paella tonight?")
```
Comment (jaylees14) [inline src/test/scala/temple/utils/StringUtilsTest.scala]:
```suggestion
    españolQue("Nosotros estamos teniendo paella esta noche") shouldEqual ("¿Nosotros estamos teniendo paella esta noche?")
```

**Review by jaylees14 — CHANGES_REQUESTED** [review]

File: src/test/scala/temple/utils/StringUtilsTest.scala (hunk 1 above)
Comment (Lewiky) [reply src/test/scala/temple/utils/StringUtilsTest.scala]: Yeah exactly this, not acceptable

**Review by Lewiky — CHANGES_REQUESTED** [review]

**Review by jaylees14 — APPROVED** [review]

**Review by Lewiky — APPROVED** [review]
si go on then

**Review by c-harding — APPROVED** [review]
Sure. What happened while I was gone?

**Pull request merged**